app/src/androidTest/java/net/skyscanner/backpack/calendar/BpkCalendarTest.kt

```diff
@@ -417,6 +417,21 @@ class BpkCalendarTest : BpkSnapshotTest() {
     snap(wrapWithBackground(calendar))
   }
 
+  @Test
+  fun screenshotTestCalendarSetSelectionFromTop_withTheme() {
```

Comment:
One last thing, can you remove `withTheme` from the name here and also not use `createThemedContext`. There is not reason to use the themed version for this test.

Reply:
Make sense, done!